test/distributed/org/apache/cassandra/distributed/test/FailingRepairTest.java

```diff
@@ -151,7 +151,11 @@ public static void setupCluster() throws IOException
                               .start());
         CLUSTER.setUncaughtExceptionsFilter((throwable) -> {
             if (throwable.getClass().toString().contains("InstanceShutdown") || // can't check instanceof as it is thrown by a different classloader
-                throwable.getMessage() != null && throwable.getMessage().contains("Parent repair session with id"))
+                (throwable.getMessage() != null && throwable.getMessage().contains("Parent repair session with id")) ||
```

Comment:
Later Guava versions seem to more vocal about exceptions so I had some extra filters for exceptions